src/Root.js

```diff
@@ -20,12 +22,23 @@ configureDevtool({
 @inject(app("UI"))
 @observer
 class Root extends React.Component {
-  render() {
+  @observable.ref
+  client = null;
+
+  async componentDidMount() {
     const {UI} = this.props;
-    const client = getClient(UI);
+    const client = await getClient(UI);
```

Comment:
I was experimenting with persisting the apollo cache. Preloading a persisted cache necessitated some async actions , and while the current code is not async I left this in for future use.